website/app/components/doc/page/banner.hbs

```diff
@@ -0,0 +1,24 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: MPL-2.0
+}}
+
+{{#if this.isVisible}}
+  <div class="doc-page-banner">
+    <FlightIcon class="doc-page-banner__icon" @name="message-circle-fill" @size={{24}} />
+    <p class="doc-page-banner__text doc-text-body">Got a minute? We’d like your feedback!
+      <a
+        href="https://docs.google.com/forms/d/e/1FAIpQLSe2tiPu_Lgcr6TIIvXSvu2RiSHU2SERW2Dpocm7P_ixNmx1BQ/viewform"
```

Comment:
```suggestion
        href="https://forms.gle/N5ibgGF3zXFFm1Lj9"
```

Reply:
@heatherlarsen give it a `yarn lint:hbs:fix` to re-arrange the markup for this link (as it got shorter it doesn't require multiple lines).